oak_attestation_verification/tests/verifier_tests.rs

```diff
@@ -181,6 +184,81 @@ fn verify_succeeds() {
     assert!(p.status() == Status::Success);
 }
 
+#[test]
+fn verify_mock_dice_chain() {
+    let mock_evidence_provider =
+        oak_restricted_kernel_sdk::mock_attestation::MockEvidenceProvider::create()
+            .expect("failed to create mock provider");
+    let mock_evidence = mock_evidence_provider.get_evidence();
+
+    let result = verify_dice_chain(
+        &evidence_to_proto(mock_evidence.clone()).expect("could not convert evidence to proto"),
+    );
+
+    assert!(result.is_ok());
+    let evidence_values: oak_proto_rust::oak::attestation::v1::extracted_evidence::EvidenceValues =
+        result.unwrap().evidence_values.unwrap();
+    assert!(matches!(evidence_values, oak_proto_rust::oak::attestation::v1::extracted_evidence::EvidenceValues::OakRestrictedKernel{..}))
```

Comment:
first assert succeeds, the second fails as the EvidenceValues is CB not `EvidenceValues`